src/sentry/static/sentry/app/views/integrationInstallation.jsx

```diff
@@ -1,112 +1,151 @@
 import React from 'react';
+import styled from 'react-emotion';
 
+import {singleLineRenderer} from 'app/utils/marked';
+import {t, tct} from 'app/locale';
+import AddIntegration from 'app/views/organizationIntegrations/addIntegration';
+import Alert from 'app/components/alert';
 import AsyncView from 'app/views/asyncView';
-import Form from 'app/views/settings/components/forms/form';
-import IndicatorStore from 'app/stores/indicatorStore';
+import Button from 'app/components/button';
+import HookStore from 'app/stores/hookStore';
 import NarrowLayout from 'app/components/narrowLayout';
-import SelectField from 'app/views/settings/components/forms/selectField';
-import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
-import {t, tct} from 'app/locale';
-
-class IntegrationInstallation extends AsyncView {
-  componentDidMount() {
-    this.dialog = null;
-    window.addEventListener('message', this.receiveMessage, false);
-  }
+import SelectControl from 'app/components/forms/selectControl';
 
-  componentWillUnmount() {
-    window.removeEventListener('message', this.receiveMessage);
+/**
+ * Currently the only integration that has a install-from-provider flow is
+ * Github, so we hardcode this to ONLY support he github provider at the
+ * moment. We'll come back later and handle more later.
```

Comment:
What about jira cloud?

Reply:
Jira cloud doesn't push you through this flow